content/en/docs/helm/helm_inspect.md

```diff
@@ -0,0 +1,7 @@
+---
+section: depreciated
+---
+
+## helm inspect
+
+This command has been depricated. Please refer instead to [helm show](../helm_show/).
```

Comment:
same comment here RE: it's an alias of `helm show` (and not actually deprecated)